### DISCUSSION TITLE
那个国内镜像网站没了导致ALAS打不开

### DISCUSSION THREAD

**Discussion by Crepus02** [LmeSzinc/AzurLaneAutoScript#2413] [Q&A] [answered]
unable to access 'https://gitee.com/lmeszinc/azur-lane-auto-script-mirror/': The requested URL returned error: 403

**Comment by kakokara** [comment]
切换为‘https://e.coding.net/llop18870/alas/AzurLaneAutoScript.git’

**Reply by jingyill** [reply]
好像也404了

**Comment by ZhiJuan0724** [comment]
直接把yaml里的autoupdate改成false

**Reply by Crepus02** [reply]
请问这个yaml里的autoupdate在哪里呀？我看到有一个yaml文件夹，但是里面都是py文件

**Reply by lyhggf** [reply]
解决了

**Comment by 18870** [comment, marked as answer]
https://github.com/LmeSzinc/AzurLaneAutoScript/issues/2412#issuecomment-1486576012
